packages/mui-base/src/InputUnstyled/useInput.types.ts

```diff
@@ -52,3 +53,50 @@ export interface UseInputInputSlotOwnProps {
 
 export type UseInputInputSlotProps<TOther = {}> = Omit<TOther, keyof UseInputInputSlotOwnProps> &
   UseInputInputSlotOwnProps;
+
+export interface UseInputReturnValue {
+  /**
+   * If `true`, the component is disabled.
```

Comment:
Hmm, I think in a situation like this, the future tense is preferable. "If X, then Y will be Z." It helps clarify cause and effect this way—Y becomes Z *because of* X. In the current version, the cause and effect are not as clear.